### ISSUE TITLE
What happened to CodeFS?

### ISSUE THREAD

**Issue by suo** [MisterTea/CodeFS#1]
I am pretty intrigued in the pitch in the README.md, but looks like there hasn't been much activity in this repo lately. I was curious what happened—any particular learnings? Technical dealbreakers? Or just not enough time?

**Comment by MisterTea** [comment]
Yep, it is functional but needs to go through a bunch of tests. My plan is to use some off the shelf POSIX filesystem tests and see if any bugs pop out. 

The remote folders in VScode is really quite good so I haven't prioritized codefs.
